tests/helpers/utils.py

```diff
@@ -111,3 +115,18 @@ def inner_f(queue, **kwargs):
         assert result == 1, 'expected 1, but returned %s' % result
 
     return wrapper
+
+
+@contextmanager
+def no_warning_call(warning_type, match: Optional[str] = None):
+    with pytest.warns(None) as record:
+        yield
+
+        try:
+            w = record.pop(warning_type)
+            if match is not None and match not in str(w.message):
```

Comment:
```suggestion
            if match is not None and match not in w.text:
```

Reply:
It threw an Error, `text` is not an Attribute of the Warning Object.

Reply:
weird, it worked for me.